lib/code_sponsor_web/views/creative_view.ex

```diff
@@ -0,0 +1,10 @@
+defmodule CodeSponsorWeb.CreativeView do
+  use CodeSponsorWeb, :view
+
+  import CodeSponsorWeb.TableView
+
+  def title(:index), do: "Code Sponsor | Creatives"
```

Comment:
Base text for this could be pushed into a private function so as to not have to write it out multiple times.